src/pool.jl

```diff
@@ -107,6 +107,64 @@ avoid doing a dict lookup twice
     i
 end
 
+function mergelevels(ordered, levels...)
```

Comment:
I have checked that this is only moved. I guess this is what you wanted but just writing to confirm.